github/src/main/kotlin/gropius/Application.kt

```diff
@@ -55,8 +55,12 @@ class Application : CommandLineRunner {
     lateinit var syncSelector: SyncSelector
 
     override fun run(vararg args: String?) {
-        runBlocking {
-            syncSelector.sync()
+        try {
+            runBlocking {
+                syncSelector.sync()
+            }
+        } finally {
+            exitProcess(0)
```

Comment:
Maybe an exit code with semantic meaning (or at least non 0 when error?)

Reply:
21296399359923d19257bbc271b41f71077fc218

Reply:
Closing and appriving asap 😉